articlequality/feature_lists/ptwiki.py

```diff
@@ -0,0 +1,160 @@
+"""
+Portuguese Wikipedia
+++++++++++++++++++++
+"""
+import re
+
+from revscoring import Feature
+from revscoring.datasources.meta import filters, mappers
+from revscoring.features import wikitext
+from revscoring.features.meta import aggregators
+from revscoring.features.modifiers import log, max, sub
+from revscoring.languages import portuguese
+
+from . import wikipedia
+
+# Templates
+infobox_templates = wikitext.revision.template_names_matching(
+    r"Info", name="ptwiki.revision.infobox_templates")
+CN_TEMPLATES = [
+    r"Carece de fontes",
+    r"Carece de fontes2",
+    r"Carece de fontes/bloco",
+    r"Carece de fontes/bloco2"
+]
+cn_templates = wikitext.revision.template_names_matching(
+    "|".join(CN_TEMPLATES), name="ptwiki.revision.cn_templates")
+main_article_templates = wikitext.revision.template_names_matching(
+    "Main", name="ptwiki.main_article_templates")
```

Comment:
Looks like the "Main" template in enwiki correspond to the "" template in ptwiki.  Here's a list of all of the templates that redirect to that "" template: 

https://pt.wikipedia.org/w/index.php?title=Especial:P%C3%A1ginas_afluentes/Predefini%C3%A7%C3%A3o:Artigo_principal&hidelinks=1&hidetrans=1

I'd change this to `r"Artigo[ _]principal|Ver[ _]artigo[ _]principal|Principal|Ver[ _]também artigo[ _]principal|Main|Detalhes|Mais|Artigoprincipal|AP|Details|Ver[ _]artigo"`